vkconfig/configurator.cpp

```diff
@@ -1198,13 +1238,30 @@ Configuration *Configurator::LoadConfiguration(const QString &path_to_configurat
         QJsonValue layer_value = layer_objects.value(layer_list[layer_index]);
         QJsonObject layer_object = layer_value.toObject();
 
-        // To match the layer we just need the name, paths are not
```

Comment:
This was a change so that VK_LAYER_PATH and user defined paths can change where the layers are pulled from. The layer paths are no longer encoded in the json files.